k3d/zarf.yaml

```
@@ -42,3 +42,13 @@ components:
         version: "###ZARF_PKG_VAR_BIGBANG_VERSION###"
         valuesFiles:
         - values/local-dev.yaml
+  - name: testing
+    actions:
+      onDeploy:
+        after:
+        - cmd: ./scripts/test-http-response.sh neuvector neuvector-service-webui 8443 
```

```suggestion
        - cmd: |
              export PATH=$(dirname ./zarf):$PATH
              ./scripts/test-http-response.sh neuvector neuvector-service-webui 8443 
```
I believe this would be required to ensure that `zarf` is on the path when that script runs, otherwise it assumes that zarf is on the user's path (not a bad assumption but better to be safe).